src/inmanta/protocol/methods.py

```diff
@@ -1,3 +1,6 @@
+"""
+Module defining the v1 rest api
+"""
```

Comment:
@arnaudsjs here!

Reply:
It seems that the license header is included in the rendered documentation and not this docstring. We should probably swap both docstrings

Reply:
![docs](https://user-images.githubusercontent.com/2684622/111489478-72466100-873a-11eb-89c1-7ad73198b34c.png)

Reply:
I think I have it now